### PR TITLE
Removes validation error when height-units is present in [widget] section

### PR DESCRIPTION
**Pull request by fi1tch**
Resolved: https://github.com/axibase/axibase-charts-vscode/issues/303

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -2,6 +2,7 @@
 {
 	"version": "0.2.0",
 	"configurations": [
+	
 		{
```
Comment (annstriganova) [inline .vscode/launch.json]: Remove extra line.

File: server/dictionary.json [hunk 2]
```diff
@@ -2987,7 +2987,7 @@
             "type": "number",
             "minValue": 0,
             "example": 2,
-            "section": "configuration",
```
Comment (rodionos) [inline server/dictionary.json]: How does this change affect validation? The `height-units` setting is allowed in [configuration], [group], and [widget] sections. In [configuration] the `height-units` setting determines the vertical unit by dividing available page height (in pixels) by the specified number. In [group], and [widget] it means something else - it determines the height of the widget in units. If we replace [configuration] with [widget] as proposed, will the validator raise error if the setting is specified in the [configuration]? Will the validation raise error if the setting is NOT present in [configuration]?

File: server/dictionary.json [hunk 3]
```diff
@@ -5424,7 +5424,7 @@
         {
             "displayName": "pointer-position",
             "widget": "chart",
-            "section": "widget",
+            "section": "series",
```
Comment (rodionos) [inline server/dictionary.json]: What's wrong with `pointer-position`?

**Comment by rodionos** [comment]
@fi1tch 

Please avoid abbreviations in PR title.
Instead of `height-un&pointer-pos sec fixed` consider `Removes validation error when height-units is present in [widget] section`

File: server/dictionary.json (hunk 3 above)
Comment (annstriganova) [reply server/dictionary.json]: The setting can be used in `[series]` section: https://apps.axibase.com/chartlab/2b05393c/2/

**Pull request merged**